### ISSUE TITLE
How to level like a boss

### ISSUE THREAD

**Issue by BogdanDBI** [HitsparkInteractive/NinOnline#93]
## Steps to reproduce issue

1.You are a lowbie that wants to level up faster ?Head to Guren/Gafuki and team up with a level 60 .
2.Let the level 60 solo the bosses till low hp then let him disconnect.
3.Last hit the bosses 

## Observed result

magically receive enormous amounts of xp that level you up

## Expected result 

 Not receive xp at all especially if you are more than 10 levels lower than the bosses

## More details

**Affected Client Version:** 

 All of them

**First noticed:**  Long ago, still works today

## Error Logs

<details><summary>View details</summary>

PASTE HERE

</details>

**Comment by RorySoh** [comment]
@michaelsakharov Right, so what we are doing is storing who did more damage in an array. But what we should be doing is calculating the damage input on the character is the majority of the HP of the NPC before giving the reward.
